src/OmniSharp.MSBuild/Analyzers/SimpleAnalyzerAssemblyLoader.cs

```diff
@@ -0,0 +1,21 @@
+﻿#if DNX451
+using System;
+using System.Reflection;
+using Microsoft.CodeAnalysis;
+
+namespace OmniSharp.MSBuild.Analyzers
+{
+    public class SimpleAnalyzerAssemblyLoader : IAnalyzerAssemblyLoader
```

Comment:
I didn't find a public implementation of it. `AnalyzerFileReference` now requires an `IAnalyzerAssemblyLoader`.